services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,

```

Comment:
Optional[list[int]]
Also the same with actors_ids

Reply:
In create_movie the same problem

Reply:
Danyl shows  in videos that "optional" was writen as I wrote through None. Can you please share more info about Optional, please?

Reply:
https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint
Also screenshot
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/65166203/216038214-83a9cf28-3af6-45c0-9a3c-bb2215e73a1d.png">